packages/angular/test/README.md

```diff
@@ -65,6 +65,14 @@ If you want to add a version-specific change, add the change inside of the appro
 
 If you need to add E2E tests that are only run on a specific version of the JS Framework, replicate the `VersionTest` component on each partial application. This ensures that tests for framework version X do not get run for framework version Y.
 
+### Testing Lazy Loaded Ionic Components
```

Comment:
This change should be reverted. It probably just got tracked between switching branches.

Reply:
Well that explains why the docs didn't show up the first time I thought I pushed to https://github.com/ionic-team/ionic-framework/pull/27867 🤦 

Fixed in https://github.com/ionic-team/ionic-framework/pull/27870/commits/0713c14368fecdd904a756e9878b06e04217fe6f